docs/messageData.md

```diff
@@ -2978,24 +3107,10 @@ With this message the organizer lets the server know to expect a federation_init
             "$comment": "public key of the remote organizer"
         },
         "challenge": {
-            "type": "object",
-            "properties": {
-                "value": {
-                    "type": "string",
-                    "contentEncoding": "hex",
-                    "pattern": "^[0-9a-fA-F]{64}$",
-                    "$comment": "A 32 bytes array encoded in hexadecimal"
-                },
-                "valid_until": {
-                    "type": "integer",
-                    "description": "[Timestamp] of the expiration time",
-                    "minimum": 0
-                }
-            },
-            "additionalProperties": false,
-            "required": ["value", "valid_until"]
-        }
-    },
+          "$ref": "../message.json",
+          "$comment": "message/message containing a FederationChallenge data"
```

Comment:
```suggestion
          "$comment": "message containing a FederationChallenge"
```

Reply:
We are using the general message found at protocol/query/method/message/message.json, where the data field contains a dataFederationChallenge